test/com/google/enterprise/adaptor/database/TupleReaderTest.java

```diff
@@ -118,6 +118,24 @@ public void testVARCHAR() throws Exception {
     assertEquals(golden, result);
   }
 
+  @Test
+  public void testNullVARCHAR() throws Exception {
```

Comment:
This test case already existed (as testNULL), so you might as well delete testNULL.